packages/remix/README.md

```diff
@@ -6,4 +6,90 @@
 
 # Official Sentry SDK for Remix
 
-This SDK is work in progress, and should not be used before officially released.
+[![npm version](https://img.shields.io/npm/v/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/remix)
+[![npm dm](https://img.shields.io/npm/dm/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/remix)
+[![npm dt](https://img.shields.io/npm/dt/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/remix)
+[![typedoc](https://img.shields.io/badge/docs-typedoc-blue.svg)](http://getsentry.github.io/sentry-javascript/)
+
+This SDK is considering experimental and in an alpha state. It may experience breaking changes. Please reach out on [GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback/concerns.
+## General
+
+This package is a wrapper around `@sentry/node` for the server and `@sentry/react` for the client, with added functionality related to Next.js.
+
+To use this SDK, init Sentry in your remix entry points for both the client and server.
+
+```javascript
+// entry.client.tsx
+
+import { useLocation, useMatches } from "@remix-run/react";
+import * as Sentry from "@sentry/remix";
+import { useEffect } from "react";
+
+Sentry.init({
+  dsn: "__DSN__",
+  tracesSampleRate: 1,
+  integrations: [
+    new Sentry.BrowserTracing({
+        routingInstrumentation: Sentry.remixRouterInstrumentation(
+            useEffect,
+            useLocation,
+            useMatches,
+        ),
+    }),
+  ],
+  // ...
+});
+```
+
```

Comment:
```suggestion

Also, wrap your Remix root, with `ErrorBoundary` to catch React errors, and `withSentryRouteTracing` to get parameterized router transactions.

```typescript
// root.tsx

import {
  Links,
  LiveReload,
  Meta,
  Outlet,
  Scripts
  ScrollRestoration,
} from "@remix-run/react";

import { ErrorBoundary, withSentryRouteTracing } from "@sentry/remix";

function App() {
    return (
      <ErrorBoundary>
        <html>
          <head>
            <Meta />
            <Links />
          </head>
          <body>
            <Outlet />
            <ScrollRestoration />
            <Scripts />
            <LiveReload />
          </body>
        </html>
      </ErrorBoundary>
  );
}

export default withSentryRouteTracing(App);

```

Reply:
Not sure if that's the "Remix way" of doing thing tbh 🤔
https://remix.run/docs/en/v1/guides/constraints#higher-order-functions

I know the docs are only talking about `action`/`loader`, but I think it's meant to also reference the component
CC/ @kentcdodds

Reply:
Considering this wrapping happens only happens once for your entire app, we can get away with the single use of a HOC - especially since the alternative is some pretty unwieldy monkey patching

Reply:
Hey @MichaelDeBoey 👋 

The challenge is to be able to get the parameterized routes for `pageload`/`navigation` transactions (they are labelled and grouped by routes in Sentry platform).

For React Router versions < 6, it's not an issue, as we can reach the `history` object, and match with the routes defined on initialization all together, without needing to wrap components.

Since React Router 6, we need hooks to get those, [so we ended up wrapping `<Routes />` component where those hooks are functional](https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/react-router/#react-router-v6).

It's similar here for Remix, except we don't have `<Routes />`, but wrapping `App` allows us to instrument all routes at once.

About the `ErrorBoundary`, we already discussed to make it a part of the final wrapper, but it's not yet implemented.

Any suggestions of alternative ways to reach parameterized routes would be great, and we can simplify both Remix and React Router 6 instrumentations.

Thanks!